src/balance/balance.service.ts

```diff
@@ -142,9 +142,9 @@ export class BalanceService implements OnApplicationBootstrap {
       const [balance = 0n, decimals = 0] = [results[index * 2], results[index * 2 + 1]]
       //throw if we suddenly start supporting tokens with not 6 decimals
       //audit conversion of validity to see its support
-      if ((decimals as number) != 6) {
-        throw EcoError.BalanceServiceInvalidDecimals(tokenAddress)
-      }
+      // if ((decimals as number) != 6) {
+      //   throw EcoError.BalanceServiceInvalidDecimals(tokenAddress)
+      // }
```

Comment:
not sure why we commented this out, do we want to support non stables like ETH?

Reply:
yes